killerbee/dev_template.py

```diff
@@ -0,0 +1,169 @@
+from typing import Optional
+from typing import Dict
+from typing import Union
+from typing import Any
+
+import os
+import time
+import struct
+import time 
+from datetime import datetime, timedelta 
+from .kbutils import KBCapabilities, makeFCS 
+from .GoodFETCCSPI import GoodFETCCSPI
+
+class APIMOTE:
+    def __init__(self, dev: str) -> None:
+
+        self._channel: Optional[int] = None
+        self._page: int = 0
+        self.handle: Optional[Any] = None
+        self.dev: str = dev
+
+        self.handle = GoodFETCCSPI()
```

Comment:
Few things probably are going to heavily rely on another thing like the import on line 12 and 22 - likely it's a usb handle here that they'd generally keep - may be a good example to set up like that?

Reply:
will make further developer instruction changes in a different PR